pdf.go

```diff
@@ -859,6 +859,45 @@ func (w *pdfPageWriter) EndTextObject() {
 	w.inTextObject = false
 }
 
+/* Extract from the pdf specification:
+When a string is written by enclosing the data in parentheses, bytes whose values are the
+same as thoseof  the  ASCII  characters  left  parenthesis  (40),  right  parenthesis  (41),
+and  back-slash  (92)  must  be  preceded  by  a  backslash  character. (...)
+These  rules  apply  to  each individual  byte  in  a  string  object, whether  the  string
+is  interpreted  by  the  text-showing operators as single-byte or multiple-byte character codes.
+*/
+type stringBytesEscaper struct {
+	w           io.Writer
+	escapeChars []byte
+}
+
+func (r stringBytesEscaper) Write(p []byte) (n int, err error) {
+	var k int
+	i := 0
```

Comment:
You could initialize `k` and `i` in the same line, `var i, k int`

Reply:
you are right